src/main/java/com/dawkastresu/medicalclinic/MedicalClinicExceptionHandler.java

```diff
@@ -0,0 +1,22 @@
+package com.dawkastresu.medicalclinic;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class MedicalClinicExceptionHandler {
+
+    @ExceptionHandler(PatientNotFoundException.class)
+    protected ResponseEntity<Object> handlePatientNotFoundException(PatientNotFoundException ex) {
```

Comment:
Zamiast zwracać po prostu message, ktory nie jest w formacie JSON to utworzmy Klase ErrorMessage i niech ta klasa Error Message zawiera pola:
message <- ten message z wyjątku
HttpStatus httpStatus
LocalDateTime occuredAt